packages/rich-text/src/formatters/test/letterSpacing.tsx

```diff
@@ -73,25 +81,33 @@ describe('Color formatter', () => {
 
   describe('stylesToCSS', () => {
     it('should ignore styles without a letter spacing style', () => {
-      const css = stylesToCSS(['NOT-LETTERSPACING', 'ALSO-NOT-LETTERSPACING']);
+      const css = stylesToCSS({
+        toArray: () => ['NOT-LETTERSPACING', 'ALSO-NOT-LETTERSPACING'],
+      } as OrderedSet<string>);
```

Comment:
Uh, that's pretty ugly, but I see it's necessary. Maybe a small util function for it?

Reply:
Ended up using `OrderedSet.of()` from `immutable` for this